robusta_krr/core/models/objects.py

```diff
@@ -22,6 +22,7 @@ class K8sObjectData(pd.BaseModel):
     namespace: str
     kind: str
     allocations: ResourceAllocations
+    metadata: dict[str, str] = {}
```

Comment:
I did a dict instead of BaseModel deliberately since will be the most maintainable due to not needing to update runner every time a new field is added only for frontend like for example adding the key "hpa_managed".